utils/log-util.js

```diff
@@ -0,0 +1,25 @@
+const isDebug = process.env.DEBUG === "true";
+const isTest = process.env.TEST === "true";
+
+const debug = (...data) => {
+  if (isDebug) console.log(data.join(""));
+};
+
+const log = (data) => {
+  console.log(data);
+};
+
+const info = (data) => {
+  if (!isTest) console.error(data.replace(/^ {8}/gm, "    "));
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/openlawteam/tribute-contracts/pullRequest?prid=8819145)
```suggestion
  if (!isTest) {console.error(data.replace(/^ {8}/gm, "    "));}
```